db-migrations/3_test3.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS data_upload (
```

Comment:
is this just a temp table while you figure it out?

Reply:
Yes